packages/babel-preset-env/test/fixtures/bugfixes-babel-8/_esmodules/stdout.txt

```diff
@@ -2,48 +2,28 @@
 
 Using targets:
 {
-  "android": "61",
-  "chrome": "61",
-  "edge": "16",
-  "firefox": "60",
-  "ios": "10.3",
-  "node": "13.2",
-  "opera": "48",
+  "android": "132",
+  "chrome": "109",
+  "edge": "132",
+  "firefox": "128",
+  "ios": "15.6",
+  "opera": "113",
   "opera_mobile": "80",
-  "safari": "10.1",
-  "samsung": "8.2"
+  "safari": "18.2",
+  "samsung": "26"
```

Comment:
These test cases are updated because in Babel 8 a single `esmodules: true` will be ignored as the browsers in the `defaults` browserslist queries all support ESM.